backend/routes/issue.js

```diff
@@ -479,12 +505,17 @@ function listIssues(req, res, next) {
 	if (req.query.shared_id) {
 		findIssue = Issue.findBySharedId(dbCol, req.query.shared_id, req.query.number);
 	} else if (req.params.rid) {
-		findIssue = Issue.findIssuesByModelName(dbCol, req.session.user.username, null, req.params.rid, projection);
+		findIssue = Issue.findIssuesByModelName(dbCol, req.session.user.username, null, req.params.rid, projection, null, ids);
 	} else {
-		findIssue = Issue.findIssuesByModelName(dbCol, req.session.user.username, "master", null, projection, null, null, req.query.sortBy);
+		findIssue = Issue.findIssuesByModelName(dbCol, req.session.user.username, "master", null, projection, null, ids, req.query.sortBy);
 	}
 
 	findIssue.then(issues => {
+
+	if (req.query.convertCoords) {
+		sortViewpointAxis(issues);
```

Comment:
that's not a very clear function naming, it sounds as if you're sorting the order.

Reply:
Renamed to `changeIssueAxis();` i think that is more suited as it is changing the axis points within an issue. 

What do you think 👍 or 👎 ?